sevenseconds/aws.py

```diff
@@ -227,9 +227,27 @@ def configure_dns(account_name, cfg):
     zone = zone['GetHostedZoneResponse']
     nameservers = zone['DelegationSet']['NameServers']
     info('Hosted zone for {} has nameservers {}'.format(dns_domain, nameservers))
+    with Action('Set up DNS Delegation..'):
+        try:
+            configure_dns_delegation(account_name, nameservers, cfg)
+        except:
+            warning('DNS Delegation not possible')
     return dns_domain
 
 
+def configure_dns_delegation(account_name, nameservers, cfg):
+    dns_conn = boto.route53.connect_to_region('eu-west-1', profile_name='adminaccount')
```

Comment:
route53 is region-independent and you need a region-name for this Function.



2015-05-27 8:46 GMT+02:00 Leo Gallucci <notifications@github.com>:

> In sevenseconds/aws.py
> <https://github.com/zalando-stups/sevenseconds/pull/12#discussion_r31107166>
> :
>
> >      return dns_domain
> >
> >
> > +def configure_dns_delegation(account_name, nameservers, cfg):
> > +    dns_conn = boto.route53.connect_to_region('eu-west-1', profile_name='adminaccount')
>
> Why is eu-west-1 hard-coded here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando-stups/sevenseconds/pull/12/files#r31107166>.
>